python/ray/rllib/envs.py

```diff
@@ -1,85 +0,0 @@
-from __future__ import absolute_import
```

Comment:
This class was spamming stdout, and wasn't really needed anyways since we log elsewhere.